<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BearingsDialog</class>
 <widget class="QDialog" name="BearingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeXtRAD Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Pedestal Bearings</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dtgEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node0_range_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 0 Range (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="node0_range_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node0_bearing_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 0 Bearing (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="node0_bearing_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node1_bearing_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 1 Bearing (deg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="node1_range_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 1 Range (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="node1_bearing_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="node1_range_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="node2_range_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="node2_bearing_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 2 Bearing (deg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="node2_range_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Node 2 Range (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="node2_bearing_Edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="dtg_Edit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>410</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="dtglabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DTG (date-time group format):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(DDhhmmZ MMYY)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BearingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BearingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
